script/Character.py

```diff
@@ -1,36 +1,45 @@
 # Program imports
 import random
 
-from script import Items
+import Items
 from Mechanics.ui_mechanics import *
 from Map_Gen.ParseMap import WorldMap
 
 
 class Character:
-    def __init__(self, name, max_health, melee_attack, magic_attack,
-                 max_mana, max_stamina, defense, luck, level=1):
+    def __init__(self, name=None, max_health=1, health=max_health, melee_attack=None, magic_attack=None,
+                 max_mana=None, max_stamina=None, defense=None, luck=None, level=1):
         self.name = name
         self.level = level
         # Stats
         self.max_health = max_health
-        self.health = self.max_health
+        self.health = health
         self.melee_damage = melee_attack
         self.magic_damage = magic_attack
         self.max_mana = max_mana
         self.max_stamina = max_stamina
         self.defense = defense
         self.luck = luck
 
-        # Inventory
+        # Inventory 
         self.equipped_weapon = Items.fist
         self.equipped_armor = None
 
+    # These classes can be used for testing for the time being. They will need to integrate combat at some point.
     def melee_attack(self):
         return random.randint(self.melee_damage // 2, self.melee_damage)
 
     def magic_attack(self):
         return random.randint(self.magic_damage // 2, self.magic_damage)
 
+    def set_health(self, new_health):
```

Comment:
'set health' sounds like you are setting health to be the value. But the logic says you are actually 'adding' that health to your current health

Reply:
This method can be used for adding and subtracting health at any point in the game play. If damage needs to calculated, a negative int will be passed in to do so. If we want to semantically do this different, changes can be made, but as it stands, set_health appropriately identifies what this function does in its current state. This will likely need to be modified as we start to form exactly what the program is going to do.

Reply:
I'm fine with having a function to add health, but I'm still not sure about the naming. 'Set' methods normally mean setting a value to x, not adding x. It would be better to call it 'add_health' or 'increment_health' or something like that so it's clear what the result will be.

Reply:
okay what do you think about "adjust" technically this will handle deduction to health as well.

Reply:
Sure